cos-nvidia-gpu-installer/installer.sh

```diff
@@ -54,6 +54,8 @@ NVIDIA_DRIVER_URL="https://us.download.nvidia.com/XFree86/Linux-x86_64/${NVIDIA_
 NVIDIA_DRIVER_MD5SUM="d60819b2e377398c7296999ab5e7c1a4"
 NVIDIA_DRIVER_PKG_NAME="NVIDIA-Linux-x86_64-${NVIDIA_DRIVER_VERSION}.run"
 
+DEVICE_PLUGIN_ENABLED=${DEVICE_PLUGIN_ENABLED:-"false"}
```

Comment:
Can you update line 30 with documentation about this.

Reply:
Done.